File: Kap15/Python/Listing43/TextHandler.py
import sys
import codecs

sys.stdout = codecs.getwriter("UTF-8")(sys.stdout.detach())

def readFile(fName):
    fp = open(fName, encoding="UTF-8")
    print(fp.read())
    fp.close()

def writeFile(fName, content):
    fp = open(fName, "w", encoding="UTF-8")
    content.encode("utf-8")
    fp.write(content)
    fp.close()

readFile("C:\\temp\\Hello.txt")
writeFile("C:\\temp\\Hello11.txt", "Hallo, Welt!\nDie 1. Zeile hat zwölf Zeichen.")

File: Kap15/PHP/Listing42/TextHandler.php
<?php

readText("C:\\temp\\Hello.txt");

writeText("C:\\temp\\Hello10.txt", "Hallo, Welt!\nDie 1. Zeile hat zwölf Zeichen.");

function readText($fName) {
    $fp = fopen($fName, "r");
    echo fread($fp, filesize($fName));
    fclose($fp);
  }
  
function writeText($fName, $content) {
    $fp = fopen($fName, "w");
    fwrite($fp, $content);
    fclose($fp);
}
  
?>